docker.ps1

```diff
@@ -0,0 +1,63 @@
+param(
+ [Parameter(Mandatory=$false)]
+ [string]$Command
+)
+$ErrorActionPreference = "Stop"
+$GIT_COMMIT = (git rev-parse HEAD)
+$GIT_BRANCH = (git rev-parse --abbrev-ref HEAD)
+$IMAGE_NAME = "rustvmm/dev"
+$REGISTRY = "index.docker.io"
```

Comment:
I have no experience with power shell, but is there a way to have a common file (e.g. `docker.env`) between `docker.sh` and `docker.ps1` where we define common variables or commands?

I mean something that in bash we can include with `source` where we define variables that we use in both, so we have one place to edit them.

I'm not sure if it's easy to do though, I found something here: 
https://stackoverflow.com/questions/72236557/how-do-i-read-a-env-file-from-a-ps1-script

But again, I have no experience, so it's up to you whether it makes sense or not.